docs/source/debugging.rst

```diff
@@ -0,0 +1,50 @@
+Debugging Errors
+================
+
+First, thank you for using Starfish and SpaceTx-Format! Feedback you provide on features and the
+user experience is critical to making Starfish a successful tool. Because we iterate quickly on this
+feedback to add new features, things change often, which can result in your code getting out of sync
+with your data. When that happens, you may observe errors.
+
+Most of the time, you can fix this problem by pulling the most recent version of the code,
+reinstalling starfish, and restarting your environment. If you're using starfish with datasets from
+spaceTx located on our cloudfront distribution, we're committed to keeping that data up to date.
+You can find it in a versioned folder, and the version will correspond to the version of starfish
```

Comment:
This is not particularly clear.

Reply:
Can we commit to this? Are there problems I'm not seeing with replacing dates with versions? I'm happy to add an example such as: 

```diff
For example, if you are upgrading from starfish 0.1.0 0.1.1, you will need to change 
references to our datasets as follows: 

- http://czi.starfish.data.public.s3.amazonaws.com/browse/formatted/0.1.0/iss_breast/experiment.json
+ http://czi.starfish.data.public.s3.amazonaws.com/browse/formatted/0.1.1/iss_breast/experiment.json
```

Reply:
This will likely result in more copies of our data.  Not every version of starfish changes the file format.

Reply:
Ah, that's a good point. What do you think is the right approach for the user experience?

Reply:
One option: it's more overhead, but we can create a top-level experiment.json that points to a bunch of shared data files.

Reply:
It seems like we may need to move in that direction anyway with spaceTx data coming. Our model doesn't seem sustainable when we move into the 100s of Tbs. With that stipulation in mind, does it make sense to go that route? 

I think the right path here is for me to remove it as you say, and for you to add an issue to the backlog to mark the need to revisit this?